packages/bits/src/lib/toolbar/toolbar.component.html

```diff
@@ -3,10 +3,11 @@
      (containerResize)="moveToolbarItems()"
      #toolbarContainer>
     <div class="nui-toolbar-content__dynamic d-flex flex-row flex-shrink-0">
-        <ng-template ngFor let-commandGroup let-last="last" [ngForOf]="commandGroups">
-            <ng-template ngFor let-commandItem [ngForOf]="commandGroup.items">
+        <ng-template ngFor let-i="index" let-commandGroup let-last="last" [ngForOf]="commandGroups">
+            <ng-template ngFor let-commandItem let-j="index" [ngForOf]="commandGroup.items">
                 <button nui-button type="button"
-                        (click)="commandItem.actionDone.emit()"
+                        [tabindex]="(!i && !j) ? 0 : -1"
```

Comment:
Just a quick question. I'm not exactly sure what this is doing. Do you mind explaining?

Reply:
as we decided don't put focus on all toolbar container, I just use firs element as a trap to catch focus. Then user could navigate by arrows. For rest of buttons I set tabindex to -1 to allow move from toolbar by pressing Tab. (!i && 1J) is not obvious and I could add comment for this if you wish. i - point to first group inside toolbar, j - to firs item inside this group.

Reply:
Thanks, Dima. I think an inline comment about that would indeed be helpful.